src/components/CreateMatchExperienceModal/index.tsx

```diff
@@ -111,42 +166,59 @@ const CreateMatchExperienceModal = ({ isOpen, onClose }: CreateMatchExperienceMo
 
     return (
         <Modal
-            title="Create MatchExperience"
+            title="Create your match experience"
             open={isOpen}
             onCancel={() => {
                 reset();
                 setImageUrl('');
                 setSelectedCountry('');
                 setSelectedLeague(undefined);
+                setSelectedDate(null);
+                setTeams([]);
                 onClose();
             }}
             footer={null}
             className={styles.modal}
         >
             <Form layout="vertical" className={styles.form} onFinish={handleSubmit(onSubmit)}>
+                <Form.Item
+                    label="Match Date"
+                    validateStatus={errors.matchDate ? 'error' : ''}
+                    help={errors.matchDate?.message}
+                >
+                    <Controller
+                        name="matchDate"
+                        control={control}
+                        render={({ field }) => (
+                            <DatePicker {...field} style={{ width: '100%' }} onChange={onDateChange} />
```

Comment:
```suggestion
        <DatePicker {...field} style={{ width: '100%' }} onChange={(date) => field.onChange(date)} />
/>
```

add the date as part of form. does not need its own state

Reply:
Why? I do need a custom handleChange because I handle other things there as well